packages/eslint-plugin/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### Breaking Changes
+
+-   `@wordpress/is-gutenberg-plugin` rule has been replaced by `@wordpress/wp-global-usage` ([#61486](https://github.com/WordPress/gutenberg/pull/61486)).
+-   `@wordpress/wp-process-env` rule has been added and included in the recommended configurations ([#61486](https://github.com/WordPress/gutenberg/pull/61486)).
```

Comment:
`@wordpress/gutenberg-phase` rule removed. That doesn't exist and hasn't been used for some time.

Reply:
Yes, it stopped being a thing after WordPress Phase 2 was concluded - sometime between WP 5.9 and 6.3 when site editor was going out of beta.